superset-frontend/src/explore/components/controls/ColorPickerControl.jsx

```diff
@@ -26,10 +26,12 @@ import ControlHeader from '../ControlHeader';
 const propTypes = {
   onChange: PropTypes.func,
   value: PropTypes.object,
+  sketchPickerWidth: PropTypes.number,
```

Comment:
not sure if you'd rather just hard code in a bigger width, rather than having it as a prop